? unxlngx4.pro
cvs diff: Diffing .
cvs diff: Diffing prj
cvs diff: Diffing source
cvs diff: Diffing source/analysis
Index: scaddins/source/analysis/analysishelper.hxx
===================================================================
RCS file: /cvs/sc/scaddins/source/analysis/analysishelper.hxx,v
retrieving revision 1.28
diff -u -r1.28 analysishelper.hxx
--- scaddins/source/analysis/analysishelper.hxx	23 Dec 2004 10:42:14 -0000	1.28
+++ scaddins/source/analysis/analysishelper.hxx	4 Jun 2005 19:30:28 -0000
@@ -397,7 +397,7 @@
 
                                 /// @return  element on position nIndex or 0 on invalid index
     inline sal_Int32            Get( sal_uInt32 nIndex ) const
-                                    { return (sal_Int32) MyList::GetObject( nIndex ); }
+                                    { return (sal_Int32)(sal_IntPtr) MyList::GetObject( nIndex ); }
 
                                 /// @return  number of elements in the range of nMinVal to nMaxVal (both included)
     sal_Int32                   CountCondition( sal_Int32 nMinVal, sal_Int32 nMaxVal ) const;
cvs diff: Diffing source/datefunc
